lib/matplotlib/ticker.py

```diff
@@ -2634,10 +2634,21 @@ def __init__(self, n=None):
 
         If *n* is omitted or None, it will be set to 5 or 4.
         """
-        self.ndivs = n
+        if n is None:
```

Comment:
This whole block is equivalent to `self.ndivs = n`. Why is it here? Did you want to have a case where that's not true?

Reply:
Sorry my bad, i left that chuck while trying to solve the problem via __init__ method. Will correct it back.